### PR TITLE
Xclip refactor

### PR DESCRIPTION
**Pull request by akallabeth**

### CODE REVIEW

File: winpr/libwinpr/clipboard/clipboard.c [hunk 1]
```diff
@@ -63,6 +63,13 @@ static const char* CF_STANDARD_STRINGS[CF_MAX] = {
 	"CF_DIBV5"         /* 17 */
 };
 
+const char* ClipboardGetFormatIdString(UINT32 formatId)
+{
+	if (formatId < ARRAYSIZE(CF_STANDARD_STRINGS))
+		return CF_STANDARD_STRINGS[formatId];
+	return "CF_REGISTERED_FORMAT";
```
Comment (pnowack) [inline winpr/libwinpr/clipboard/clipboard.c:70]: Wouldn't `CF_UNREGISTERED_FORMAT` be better (since the string to look for could not be found)?

Comment (akallabeth) [reply winpr/libwinpr/clipboard/clipboard.c:70]: the formats not predefined must be (manually) registered, so that is the reason for the name

Comment (pnowack) [reply winpr/libwinpr/clipboard/clipboard.c:70]: ah ok

**Review by pnowack — APPROVED** [review]
Could not spot something wrong here.

**Pull request merged**